Dockerfile

```diff
@@ -64,8 +69,6 @@ WORKDIR $TWLIGHT_HOME
 
 COPY manage.py /app/manage.py
 
-# Configure static assets.
-RUN SECRET_KEY=twlight /app/bin/twlight_static.sh
```

Comment:
At some point we had to have this in the image to make our ci process happy; We should go have a look through the github actions for pull requests and production branch merging before making changes.

Reply:
Yes, I'd love to look into how the CI/CD process flows!